<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>QSerialPortTestClass</class>
	<widget class="QMainWindow" name="QSerialPortTestClass">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>600</width>
				<height>400</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>QSerialPortTest</string>
		</property>
		<widget class="QWidget" name="centralWidget">
			<widget class="QLabel" name="label">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>20</y>
						<width>47</width>
						<height>13</height>
					</rect>
				</property>
				<property name="text">
					<string>Port:</string>
				</property>
			</widget>
			<widget class="QComboBox" name="portComboBox">
				<property name="geometry">
					<rect>
						<x>70</x>
						<y>20</y>
						<width>150</width>
						<height>22</height>
					</rect>
				</property>
			</widget>
			<widget class="QLabel" name="portStatusLabel">
				<property name="geometry">
					<rect>
						<x>250</x>
						<y>20</y>
						<width>150</width>
						<height>16</height>
					</rect>
				</property>
				<property name="text">
					<string>Status: Port fermé</string>
				</property>
			</widget>
			<widget class="QPushButton" name="openPortButton">
				<property name="geometry">
					<rect>
						<x>430</x>
						<y>20</y>
						<width>120</width>
						<height>30</height>
					</rect>
				</property>
				<property name="text">
					<string>Ouvrir le port</string>
				</property>
			</widget>
			<widget class="QPlainTextEdit" name="receivedDataText">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>70</y>
						<width>550</width>
						<height>150</height>
					</rect>
				</property>
			</widget>
			<widget class="QLabel" name="labelLatitude">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>240</y>
						<width>100</width>
						<height>20</height>
					</rect>
				</property>
				<property name="text">
					<string>Latitude :</string>
				</property>
			</widget>
			<widget class="QLabel" name="latitudeLabel">
				<property name="geometry">
					<rect>
						<x>120</x>
						<y>240</y>
						<width>150</width>
						<height>20</height>
					</rect>
				</property>
				<property name="text">
					<string>0.000000</string>
				</property>
			</widget>
			<widget class="QLabel" name="labelLongitude">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>270</y>
						<width>100</width>
						<height>20</height>
					</rect>
				</property>
				<property name="text">
					<string>Longitude :</string>
				</property>
			</widget>
			<widget class="QLabel" name="longitudeLabel">
				<property name="geometry">
					<rect>
						<x>120</x>
						<y>270</y>
						<width>150</width>
						<height>20</height>
					</rect>
				</property>
				<property name="text">
					<string>0.000000</string>
				</property>
			</widget>
			<widget class="QLineEdit" name="messageLineEdit">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>310</y>
						<width>450</width>
						<height>30</height>
					</rect>
				</property>
			</widget>
			<widget class="QPushButton" name="sendButton">
				<property name="geometry">
					<rect>
						<x>490</x>
						<y>310</y>
						<width>80</width>
						<height>30</height>
					</rect>
				</property>
				<property name="text">
					<string>Envoyer</string>
				</property>
			</widget>
			<!-- Nouveau bouton pour envoyer les coordonnées GPS -->
			<widget class="QPushButton" name="sendCoordinatesButton">
				<property name="geometry">
					<rect>
						<x>490</x>
						<y>350</y>
						<width>80</width>
						<height>30</height>
					</rect>
				</property>
				<property name="text">
					<string>Envoyer Coord</string>
				</property>
			</widget>
		</widget>
	</widget>
	<resources/>
	<connections/>
</ui>
